dts/bindings/gpio/ite,it8xxx2-gpiokscan.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2022 ITE Corporation. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+description: This binding gives a base representation of the ITE kscan pins as gpio
+
+compatible: "ite,it8xxx2-gpiokscan"
+
+include: [base.yaml]
+
+properties:
+  reg:
+    required: true
+
+  ngpios:
+    type: int
+    required: true
+    default: 8
```

Comment:
Default values must be described (see other bindings).

Reply:
I removed default value.